backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresConfiguration.java

```diff
@@ -70,6 +74,7 @@ public static class Builder {
         private Optional<String> nonRLSUser = Optional.empty();
         private Optional<String> nonRLSPassword = Optional.empty();
         private Optional<Boolean> rowLevelSecurityEnabled = Optional.empty();
+        private Optional<String> sslMode = Optional.empty();
```

Comment:
Can this be an enum so that values are explicit?

Reply:
SSLMode (from r2dbc, that ends up being what is used in our PostgresConfiguration after build) at the end is an enum.

Enough?

Reply:
Enough